src/Common/src/Interop/Unix/System.Native/Interop.DlOpen.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [Flags]
+        internal enum DlOpenFlags : int
+        {
+            RTLD_LAZY = 1,
+            RTLD_NOW = 2,
+            RTLD_GLOBAL = 4,
+            RTLD_LOCAL = 8
+        }
+
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_DlOpen")]
+        internal static extern IntPtr DlOpen(string fileName, DlOpenFlags flag);
```

Comment:
It looks like dlopen is also be used in tests, e.g.
https://github.com/dotnet/corefx/blob/4bf90198c5e7710ea67b7c1b467e859ec8bb78e5/src/System.Drawing.Common/tests/Helpers.cs#L37
Do those need to be fixed?

Reply:
I updated both usages of `dlopen`. I couldn't find any other places that use `dlopen` so I think we're good now.